.phan/stubs/wpcom-stubs.php

```diff
@@ -1435,6 +1435,12 @@ public static function site_requires_upgrade($blog_id)
         public static function get_costs()
         {
         }
+        /**
+         * @param int $blog_id
+         * @return string
+         */
+        public static function get_upgrade_url($blog_id){
```

Comment:
As noted at the top of this file, the manual addition of the stub here is being overwritten by #37765 because it wasn't added to fbhepr%2Skers%2Sjcpbz%2Sova%2Sgrnzpvgl%2Qohvyqf%2Swrgcnpx%2Qfghof%2Sfgho%2Qqrsf.cuc%3Se%3Q2q082p6q%23162-og.

Reply:
Thank you for the comment. Going to update the `stubs-defs.php` now.

Reply:
Change is ready for reviews. Going to land it and run the TeamCity job right after.

Reply:
Should be fixed now.